Examples/Profiles/Profile-example.fsh

```diff
@@ -1,4 +1,39 @@
 // @Name: Profile Example
+// @Description: Example of a profile of the Patient resource. This example includes a few of the most commonly used constraints and documentation features of FHIR profiles.
 
-// Nothing yet! 
-// Help wanted -- your example can go here. 
\ No newline at end of file
+Profile:     ExamplePatientProfile
+Id:          example-patient-profile
+Parent:      Patient
+Title:       "Example Patient Profile"
+Description: "Example of a profile of Patient"
+
+// The contents of `^short` appears in the \"Description & Constraints\" column in the differential or snapshot tables of a built Implementation Guide.
+// In this case, `. ^short` sets this for the root level of the element hierarchy displayed in the table. `someElement ^short` will set this text for a specific
+// element, as is seen below.
```

Comment:
join this with the previous line.

Reply:
I reflowed all the comments in this file so that they hard break at around 100 chars. This should make them much easier to read as FSHOnline does not appear to soft wrap.